cli/src/alluxio.org/cli/cmd/job/cmd_status.go

```diff
@@ -0,0 +1,61 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package job
+
+import (
+	"strconv"
+
+	"github.com/palantir/stacktrace"
+	"github.com/spf13/cobra"
+
+	"alluxio.org/cli/cmd"
+	"alluxio.org/cli/env"
+)
+
+var CStatus = &CStatusCommand{
```

Comment:
```suggestion
var CmdStatus = &CmdStatusCommand{
```

better to expand the name, especially for public variables